### DISCUSSION TITLE
Handling Hover, Focus, and Other States does nothing in my partial template

### DISCUSSION THREAD

**Discussion by MikiVanousek** [tailwindlabs/tailwindcss#7865] [Help]
I added `print:hidden` attribute into my partial template, and it does not seem to work. Even when pasting [the example](https://tailwindcss.com/docs/hover-focus-and-other-states#print-styles) into my template, 
```
<div>
  <article class="print:hidden">
    <h1>My Secret Pizza Recipe</h1>
    <p>This recipe is a secret, and must not be shared with anyone</p>
    <!-- ... -->
  </article>
  <div class="hidden print:block">
    Are you seriously trying to print this? It's secret!
  </div>
</div>
```
the attributes behind `print:` are never triggered. I also noticed that other states like 'hover:` also seems to do nothing.

**Comment by GeorgeFlorian** [comment]
Have you ever fixed it ? I am encountering the same issue

**Reply by wongjn** [reply]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The class name would resolve to a Tailwind class that Tailwind would generate a rule for.

Otherwise, consider providing a project that reproduces the unexpected behavior.

**Reply by MikiVanousek** [reply]
It has been a while, but I think properly recompiling Tailwind fixed it.
